views/appeal-application/task-list.njk

```diff
@@ -27,7 +27,7 @@
                         }) }}
                     {% endif %}
                 </li>
-                <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+                <li><hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible"></li>
```

Comment:
please don't add a <li> just for the <hr> that is not semantically correct. Instead the <li> should have a css class with the proper styling like borders, paddings and/or margins